cmd/juju/application/config.go

```diff
@@ -162,6 +172,13 @@ func (c *configCommand) Init(args []string) error {
 	}
 }
 
+func (c *configCommand) validateGeneration() error {
+	if !set.NewStrings("", "current", "next").Contains(c.generation) {
```

Comment:
Is this correct, it should contain `""` ?

Reply:
Yeah, if not passed at the command line at all, it will be empty. Normal option parsing prevents using `--generation` without a value.